pages/validators/upgrades/_meta.json

```diff
@@ -1,5 +1,6 @@
 {
     "types_of_upgrades": "Types of Upgrades",
     "performing_upgrades": "Performing Upgrades",
-    "cosmovisor": "Cosmovisor"
+    "cosmovisor": "Cosmovisor",
+    "using_cosmovisor_to_stage_dYdX_Chain_binary_upgrade": "Using Cosmovisor to stage dYdX Chain binary upgrade"
```

Comment:
need to add `how_to_set_up_a_full_node` page

Reply:
Do we want both of them in the upgrades section? I added under the other page under pages/validators/_meta.json.

Reply:
Oops missed that, LGTM then